packages/url_launcher/lib/url_launcher.dart

```diff
@@ -30,10 +30,17 @@ const MethodChannel _channel =
 ///
 /// Note that if any of the above are set to true but the URL is not a web URL,
 /// this will throw a [PlatformException].
+///
+/// [statusBarBrightness] is only used in iOS. Sets the status bar brightness
+/// of the application after opening a link. The previous value will be
```

Comment:
Add more details to 'previous value will be automatically restored'. Describe that it's stored on the platform side.

Reply:
Expanded to mention that it is stored on the platform side and restored when returning